### ISSUE TITLE
Document input methods

### ISSUE THREAD

**Issue by travankor** [void-linux/void-docs#261]
On GNOME this should be out of the box although I have never used GNOME. There are different ways this functions on Wayland vs X iirc. The popular ones are `fcitx` and `ibus`. IIRC, there are different "backends" for some languages.

c.f https://old.reddit.com/r/voidlinux/comments/ger0ju/unable_to_get_ibus_to_work_in_void/

**Comment by ericonr** [comment]
The solution they found seems to be the way the Arch wiki suggests launching the application, so I'm not sure how necessary it is. I suggested they open a request for the package they need for their prefered method of international input. I'm not sure how much of this fits in the docs.

**Comment by ericonr** [comment]
Do you have an idea for what you'd put in such a section? I'm pending towards it not fitting here.

**Comment by travankor** [comment]
I'm not very knowledgeable in this subject, honestly, so I can't answer. This seemed like an accessibility issue, so I thought it would be worth mentioning. It's mostly useful for languages that use Hanzi (Chinese characters), since there are too many to fit on a keyboard (although any language can be used).

Maybe a list of what Void offers as accesiblity tooling will suffice?

**Comment by sgn** [comment]
I'm Vietnamese, using `fcitx` and `fcitx-unikey` (I maintain `fcitx-unikey`)
because it's easier and the IME is more familiar to us, despite Vietnamese script is Latin-based.

I starts `fcitx` with `.xprofile` and export necessary environment variable.

XFCE4 and Mate (and maybe Cinnamon) seems to start `ibus` and `fcitx` automatically.

Pinging @ndgnuh, who is the other Vietnamese and actively maintains `ibus-bamboo`

**Comment by ndgnuh** [comment]
> Maybe provide some general information on accessibility tooling will suffice? 

I agree with this, maybe we should provide info such as  which package to install (since  distributions has different naming, not having to looking for package would be nice imo) and maybe some gotchas if there's any?

> The solution they found seems to be the way the Arch wiki suggests launching the application, so I'm not sure how necessary it is.

Yeah, there will be some information overlap. [The manual](https://docs.voidlinux.org/about-handbook/index.html) has also stated something like this:
>Additionally, the Arch Wiki provides a fairly comprehensive outline of common Linux software configuration, and a variety of internet search engines are available for further assistance.

I don't know if we should take into account that some people won't bother reading the `about` section (I did this :shame:) and just search what they need though.

That aside, I'd love to write/translate documentation from `ibus-bamboo` if that's what we are doing.

**Comment by ericonr** [comment]
In that interest, I'm not against a small section with these specifics. I can take a look at where it would fit exactly, but it would be awesome if one of you could do it, since you have experience with them.

We could mention the existence of the tools ("search for packages named `ibus` or `fcitx`, they are available for these languages..." - if the package name + short_desc isn't enough to determine what language/alphabet the package is for, we could add a bit more detail as well, though I'd prefer it it could be fixed in void-packages) and how to start them in the broadest terms.

@ndgnuh don't worry, I've skipped reading some parts of the handbook at some points too :p it happens! Regarding `ibus-bamboo`, do you know how active upstream is? I feel like it would be better to have better official docs there, specially because this makes any contribution have an even wider reach.

**Comment by ndgnuh** [comment]
[Upstream](https://github.com/BambooEngine/ibus-bamboo)'s last commit is 19 days ago, so I'd say it's still active.

**Comment by ericonr** [comment]
So I'd say most docs, if possible, could come from there, and we can work on Void specific stuff here. Possibly (probably) link to the project page as well.

**Comment by st3r4g** [comment]
Leaving here the steps for gnome 40 + ibus + mozc:
1. install `mozc ibus-mozc`
2. in Setting > Keyboard > Input Sources, add Japanese (Mozc)
3. (non-trivial) set `active_on_launch: True` in `~/.config/mozc/ibus_config.textproto` (otherwise mozc starts in direct input mode) found [here](https://github.com/google/mozc/issues/201)
4. `ibus write-cache; ibus restart`
by default Super+Space toggles IME on.
